kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/AccordionContainer.vue

```diff
@@ -42,28 +42,24 @@
     },
     data() {
       return {
-        itemIds: [],
+        expandedItemIds: [],
         optionsIdList: [],
       };
     },
     methods: {
       toggleItemState(id) {
-        const index = this.itemIds.indexOf(id);
+        const index = this.expandedItemIds.indexOf(id);
         if (index === -1) {
-          this.itemIds.push(id);
+          this.expandedItemIds.push(id);
         } else {
-          this.itemIds.splice(index, 1);
+          this.expandedItemIds.splice(index, 1);
         }
       },
       isItemExpanded(id) {
-        if (this.itemIds.includes(id)) {
-          return true;
-        } else {
-          return false;
-        }
+        return this.expandedItemIds.includes(id);
       },
       isOptionSelected(optionId) {
```

Comment:
this is absolutely a nitpick and non-blocking, but if an 'option' and an 'item' are functionally the same thing (i.e. a selectable accordion and its contents) then I think it would be helpful to have consistency here, and always name related functions with the same reference. So, for example `isItemExpanded` and `isItemSelected`. If these are used for _different_ parts of the accordion container, then I think having more verbose names would be helpful, just for clarity and readability.